resolve issue with header stories throwing errors

M0nica opened the pull request:
Resolve: https://github.com/NYPL/nypl-design-system/issues/123 

## **This PR does the following:**
- Add React.fragments around the bold and link header examples in order to prevent multiple child error from being thrown 

### Front End Review:
- [ ] View https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/heading--headings-with-bold
- [ ] View https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/heading--headings-with-link

M0nica committed:
resolve issue with header stories throwing errors

M0nica committed:
undo formatting changes to Heading.tsx

helenvholmes reviewed (changes requested):
Nice, so great to not have these throwing errors.

@M0nica, we have such a large number of components that we'd like to flatten @hokei and I talked about moving things up to `src/react-components/` as we touched any particular file (like you did for #137). Can you:

- Move `Heading` up to the top level as part of this PR and
- Update the package.json and CHANGELOG?

Thank you!

M0nica committed:
Merge branch 'development' of github.com:NYPL/nypl-design-system into 123fixHeaderErrors

M0nica committed:
mv `Headings/*` into top-level + update changelog

M0nica commented:
> Nice, so great to not have these throwing errors.
> 
> @M0nica, we have such a large number of components that we'd like to flatten @hokei and I talked about moving things up to `src/react-components/` as we touched any particular file (like you did for #137). Can you:
> 
>     * Move `Heading` up to the top level as part of this PR and
> 
>     * Update the package.json and CHANGELOG?
> 
> 
> Thank you!

👍 I just updated to move the `Headings/` from `/02-molecules` into top-level `/components` and updated the CHANGELOG accordingly.

helenvholmes reviewed (approved):
Thanks @M0nica, LGTM!

hokei commented on src/react-components/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "0.3.6",
+  "version": "0.3.7",
```
A note:  There's no need to bump this every time, just after a release.  The last release was `0.3.4` so we should be at `0.3.5` (with an implied prerelease)

helenvholmes replied:
Whoops, that was me requesting this. Sorry!
